apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -197,7 +224,48 @@ NetSimRouterLogModal.prototype.isVisible = function () {
  * Fill the root div with new elements reflecting the current state
  */
 NetSimRouterLogModal.prototype.render = function () {
-  // Be lazy, don't render if not visible.
+  if (usingNewLogBrowser()) {
+    this.newRender_();
+  } else {
+    this.oldRender_();
+  }
+};
+
+NetSimRouterLogModal.prototype.newRender_ = function () {
```

Comment:
The reason for the experiment is this: We want to enable the new log browser (with the new teacher view) at about the same time we send out a CSP update announcing the changes (along with some other changes) - so there will be a delay in turning this on.  However, we want to be able to take updated screenshots/videos with the new versions before that announcement.  At the very latest, we should be permanently switching to the new mode by mid-autumn, and will be able to remove the old mode (and maybe do some awesome refactoring) then.